code/game/objects/items/storage/boxes.dm

```diff
@@ -636,47 +638,59 @@
 /obj/item/storage/box/nade_box/update_icon()
```

Comment:
Also of note, not that I'm sure what can easily be done about this, but moving towards overlays instead means the vendor icon previews now don't have the labels. This should be what creates them: https://github.com/cmss13-devs/cmss13/blob/1beda3b48be1a2bf7f6453a65460feded955b55c/code/modules/asset_cache/asset_list_items.dm#L283

Old: 
![image](https://github.com/user-attachments/assets/2340323f-1a46-4c30-a297-74752f0328c2)

New:
![image](https://github.com/user-attachments/assets/ff527154-3b24-4778-b69a-22961ee831dd)

Reply:
I cannot do anything overly complex with it, i can look how similar lists are done in other places, but i can also just go bootleg way and create .dmi with preview icons that'd be used here instead

Reply:
I mean it works, i'm just not sure how acceptable it is to have buffer dmi files
![HjuMIJmv9O](https://github.com/user-attachments/assets/0401987a-4809-496a-bc60-1029cb824080)

Reply:
I don't know exactly what you mean so I cannot evaluate it. It's just that the sprites used to be static, now they're formed with overlays, and the Sprite sheet generation for these items don't incorporate the overlays when flattened into the spritesheet

Reply:
I've made a separate dmi with pre-assembled icons for ammo boxes and grenade boxes, since current system doesn't accept overlays easily, i can just continue doing it if it's acceptable way of solving it

Reply:
It would be instead of overlays; but whatever way you want. IMO though I think developing a way for spritesheets to flatten their overlays would be a good optional thing to have though. If you aren't sure how to do it, I can see about fiddling with it.

Reply:
Yeah it would be nice, but I couldn't find good examples of it on other servers either, I'll pr my variant, if you'd come up with better solution in the future, I'll implement it for vendors instead